assets/src/containers/AssignmentPlanning.js

```diff
@@ -66,6 +66,15 @@ function AssignmentPlanning (props) {
     />
   )
 
+  const tableBuilder = (loaded, assignmentData, table) => {
```

Comment:
`table` param isn't used - could you please remove it?

Reply:
Yep, removed